docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,22 @@
+# Tutorial: deploy the gateway API integrator charm
```

Comment:
Nit-pick: Capitalize "Gateway" (maybe also "integrator"? It's a bit unclear to me whether "integrator" also needs to be capitalized though...)

Reply:
I think since integrator is a generic term that signifies a charm that helps with integration (c.f nginx-ingress-integrator), it should not be capitalized. I have capitalized Gateway though.